docs/user-guide/federation/replicasets.md

```diff
@@ -0,0 +1,105 @@
+---
+---
+
+This guide explains how to use replicasets in Federation control plane.
```

Comment:
Also, "Federation control plane" -> "the Federation control plane" (or "a Federated control plane.")